docs/theory/Validation_pipe_diameter_sizing.rst

```diff
@@ -0,0 +1,1148 @@
+
+    :math:`C_{capex}`: the capital expenditure of the whole system [€].
+
+    :math:`C_{opex}`: the yearly operational expenditure of the whole system [€/y].
+
+    :math:`t_{lifetime}`: the lifetime of the whole system [y].
+
+ 
+In the current version, the routine take the one-year time series provided by the user and assumes that it holds for every year
+in the modeled systems' lifetime of 30 years. This is the default value, which is not changeable by the user in the current version and it also holds for all assets in a network. 
+It then multiplies the fixed operational cost with the number of years for which the system is optimized (i.e. 30 years).
+Note that this version of the optimizer routine does not consider any discount rate over the lifetime of the system. 
+
+The computed costs by the optimizer routine can be displayed in the KPI dashboard of the ESDL Mapeditor. 
+:numref:`KPI_dashboard` shows an example.
+
+
+.. _KPI_dashboard:
+
+.. figure:: ../images/KPI_dashboard.png
+    :figwidth: 6.94792in
+    :align: center
+
+    Screenshot of the KPI dashboard.
+
+
+The optimizer routine only considers the following costs per asset type:
+
+
+
++-----------------------------+----------------------+
+| Asset                       | Cost Coefficients    |
++=============================+======================+
+|| HeatStorage                || variable opex costs |
+|                             || fixed opex costs    |
+|                             || investment costs    |
++-----------------------------+----------------------+
+| Pipe                        || investment costs    |
+|                             || installation costs  |
++-----------------------------+----------------------+
+| HeatingDemand               || investment costs    |
+|                             || installation costs  |
++-----------------------------+----------------------+
+| Source                      || variable opex costs |
+|                             || fixed opex costs    |
+|                             || investment costs    |
++-----------------------------+----------------------+
+
+Here it is important to note that
+
+* variable opex costs need to be specified in €/Wh.
+* fixed opex costs of an asset, which are the cost made independently of the operation of the asset, need to be specified in €/W.
+* investment costs of an asset in €/W.
+* installation cost are one time costs for an asset in €.
+
+The optimizer routine computes the total installation costs of the whole system, :math:`C_{install,tot}`, as:
+
+.. _eq_install_cost:
+
+.. math::
+    :label: install_cost
+
+    C_{install,tot} = C_{install,pipe_{sup}} + C_{install,demand} + C_{install,source}
+
+In which:
+
+    :math:`C_{install,pipe_{sup}}`: the installation cost of the supply pipe(s) [€].
+
+    :math:`C_{install,demand}`: the installation cost of the heating demand(s) [€].
+
+    :math:`C_{install,source}`: the installation cost of the source(s) [€].
+
+Note that 
```

Comment:
In a future version of the documentation, we could include references to the specific parts of the code

Reply:
Yes, I think we should have two documentations. This documentation is for "interface" users. We should also make a developers documentation.

Reply:
Agree!